src/20241104_AaveV3EthereumLido_OnboardGHOAndMigrateStreamsToLidoInstance/AaveV3EthereumLido_OnboardGHOAndMigrateStreamsToLidoInstance_20241104.sol

```diff
@@ -0,0 +1,209 @@
+    if (meritAllownace > 0) {
+      (address aTokenAddress, , ) = AaveV3EthereumLido
+        .AAVE_PROTOCOL_DATA_PROVIDER
+        .getReserveTokensAddresses(AaveV3EthereumAssets.GHO_UNDERLYING);
+      AaveV3EthereumLido.COLLECTOR.approve(AaveV3EthereumAssets.GHO_UNDERLYING, MERIT_MULTISIG, 0);
+      AaveV3EthereumLido.COLLECTOR.approve(aTokenAddress, MERIT_MULTISIG, meritAllownace);
+    }
+
+    AaveV3EthereumLido.COLLECTOR.depositToV3(
+      CollectorUtils.IOInput({
+        pool: address(AaveV3EthereumLido.POOL),
+        underlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+        amount: 1_000e18
+      })
+    );
+
+    // swap
+    // usdt
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV3EthereumAssets.USDT_UNDERLYING,
+        amount: A_USDT_WITHDRAW_AMOUNT
+      }),
+      MiscEthereum.AAVE_SWAPPER
+    );
+    AaveV3Ethereum.COLLECTOR.withdrawFromV3(
+      CollectorUtils.IOInput({
+        pool: address(AaveV3Ethereum.POOL),
+        underlying: AaveV3EthereumAssets.USDT_UNDERLYING,
+        amount: A_ETH_USDT_WITHDRAW_AMOUNT
+      }),
+      MiscEthereum.AAVE_SWAPPER
+    );
+
+    AaveSwapper(MiscEthereum.AAVE_SWAPPER).swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV3EthereumAssets.USDT_UNDERLYING,
+      AaveV3EthereumAssets.GHO_UNDERLYING,
+      AaveV3EthereumAssets.USDT_ORACLE,
+      GHO_USD_FEED,
+      SELF,
+      IERC20(AaveV3EthereumAssets.USDT_UNDERLYING).balanceOf(MiscEthereum.AAVE_SWAPPER),
+      100
+    );
+
+    // usdc
+    AaveV3Ethereum.COLLECTOR.withdrawFromV3(
+      CollectorUtils.IOInput({
+        pool: address(AaveV3Ethereum.POOL),
+        underlying: AaveV3EthereumAssets.USDC_UNDERLYING,
+        amount: A_ETH_USDC_WITHDRAW_AMOUNT
+      }),
+      MiscEthereum.AAVE_SWAPPER
+    );
+
+    AaveSwapper(MiscEthereum.AAVE_SWAPPER).swap(
+      MILKMAN,
+      PRICE_CHECKER,
+      AaveV3EthereumAssets.USDC_UNDERLYING,
+      AaveV3EthereumAssets.GHO_UNDERLYING,
+      AaveV3EthereumAssets.USDC_ORACLE,
+      GHO_USD_FEED,
+      SELF,
+      IERC20(AaveV3EthereumAssets.USDC_UNDERLYING).balanceOf(MiscEthereum.AAVE_SWAPPER),
+      100
+    );
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: AaveV3EthereumAssets.GHO_UNDERLYING,
+      assetSymbol: 'GHO',
+      priceFeed: AaveV3EthereumAssets.GHO_ORACLE,
+      enabledToBorrow: EngineFlags.ENABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 0,
+      liqThreshold: 0,
+      liqBonus: 0,
+      reserveFactor: 10_00,
+      supplyCap: 20_000_000,
+      borrowCap: 2_500_000,
+      debtCeiling: 0,
+      liqProtocolFee: 10_00,
```

Comment:
This change does not appear on the diff. Maybe there is a need to run the tests again to regenerate the diff.